Arm/Insts/Common.lean

```diff
@@ -368,4 +368,81 @@ def elem_set (vector : BitVec n) (e : Nat) (size : Nat)
   have h : hi - lo + 1 = size := by simp only [hi, lo]; omega
   BitVec.partInstall hi lo (h ▸ value) vector
 
+----------------------------------------------------------------------
+
+-- Field unsigned, round and accumulate are optional for left shifts
```

Comment:
In reality, "optional" means:
- "can be used" or 
- "are not used" and that's why they're set to optional in Lean?

Reply:
Here it means "are not used" in the case of left shifts.